en/docs/how-tos/overriding-endpoint-information/overriding-endpoints-for-developer-first-approach-apis.md

```diff
@@ -24,63 +51,49 @@ note:
 ```
 
 ``` java tab="Example"
-bash gateway -e myEndpoint_prod_endpoint_0="http://wso2.com" -e myEndpoint_prod_basic_password=123456 /home/user/petstore-project/target/petstore-project.balx
+gateway /home/user/petstore-project/target/petstore-project.jar --myEndpoint_prod_endpoint_0="http://wso2.com" --myEndpoint_prod_basic_password=123456 
+```
+
+
```

Comment:
Same here

Reply:
This is because to keep the existing formatting as it is